src/controllers/monster.js

```diff
@@ -885,6 +888,114 @@ class Monster extends Controller {
 						}
 
 						const templateType = (data.iv === -1) ? 'monsterNoIv' : 'monster'
+						
+						sendscout: if (data.iv === -1 && this.config.general.scoutURL) {
```

Comment:
This is not a loop, so labelling and break below will not do anything.

Reply:
This contradicts the docs.